pkgs/tools/networking/curl/default.nix

```diff
@@ -62,6 +62,9 @@ stdenv.mkDerivation (finalAttrs: {
 
     # Affected versions: 7.84.0 to and including 8.1.2
     ./CVE-2023-32001.patch
+
+    # Affected versions: 7.84.0 to and including 8.1.2
```

Comment:
The range is off.

> Affected versions: libcurl [7.84.0](https://curl.se/docs/vuln-7.84.0.html) to and including [8.2.1](https://curl.se/docs/vuln-8.2.1.html)

It is probably preferable to reference https://curl.se/docs/CVE-2023-38039.html instead.

Reply:
Ah, good old Zahlendreher. Changed the other comment above as well.